### PR TITLE
backend with added new features

### PR DESCRIPTION
**Pull request by nipunasamaraweera**
all the work done by the mobile team to the backend

### CODE REVIEW

File: app_Backend/.DS_Store
Comment (MorgaineBarter) [inline app_Backend/.DS_Store:1]: this file shouldn't be included

File: app_Backend/backend_server/test.py
Comment (MorgaineBarter) [inline app_Backend/backend_server/test.py:1]: this file should also not be included

**Review by MorgaineBarter — CHANGES_REQUESTED** [review]
please remove the .DS_Store file from this contribution.

14 of 70 files changed are actual changes. I know this has taken more time than expected to clean up. would prefer to only see the files changed. please make sure this does not happen again.

**Review by nipunasamaraweera — COMMENTED** [review]
Deleted the DS_Store.

**Review by saminadhikari — COMMENTED** [review]
~Can you comment on what changes on which file. I mean using comment feature of github and not on code itself.~ Let's go with the merge and we could take care of minor issues later

~Also I noticed typo on folder `regestration`.~

**Review by MorgaineBarter — APPROVED** [review]
lets go for solo PR's from now on ok. will require more communication from the team as to which files they are working on or if entirely new (which will not cause conflicts to add)

**Pull request merged**